src/test/regress/sql/single_node.sql

```diff
@@ -906,6 +906,60 @@ WITH cte_1 AS
 SELECT bool_and(z is null) FROM cte_1;
 
 RESET citus.local_copy_flush_threshold;
+
+RESET citus.local_copy_flush_threshold;
+
+CREATE OR REPLACE FUNCTION coordinated_transaction_uses_2PC()
+RETURNS BOOL LANGUAGE C STRICT VOLATILE AS 'citus',
+$$coordinated_transaction_uses_2PC$$;
```

Comment:
These tests are not strictly necessary with this patch, as it was relevant to the previous patch. But still I didn't remove, just failing over to local host is something that we should be careful and having some extra tests sounds OK to me